### ISSUE TITLE
Listview scroll event conflicts with FlutterOverlayWindow drag,

### ISSUE THREAD

**Issue by sunjiaming** [X-SLAYER/flutter_overlay_window#104]

**Comment by Ahmadu-Suleiman** [comment]
I'm having the same issue as well

**Comment by sunjiaming** [comment]
@Ahmadu-Suleiman @X-SLAYER  Did you solve it？😊

**Comment by Ahmadu-Suleiman** [comment]
Yeah, I change the overlay flag anytime I want to scroll the list view

**Comment by sunjiaming** [comment]
@Ahmadu-Suleiman  Can you give us a minimal case？🙏

**Comment by Ahmadu-Suleiman** [comment]
You just need to set the `enableDrag` parameter to `false`, this will remove drag ability but will allow you to scroll your list view, I'm not on my computer to provide a concrete example but you can easily play around with the methods that allow you to change the `enableDrag` value.
